pkg/apis/v1beta1/ec2nodeclass.go

```diff
@@ -85,6 +85,9 @@ type EC2NodeClassSpec struct {
 	// +kubebuilder:validation:MaxItems:=50
 	// +optional
 	BlockDeviceMappings []*BlockDeviceMapping `json:"blockDeviceMappings,omitempty"`
+	// InstanceStorePolicy describes how to format and mount instance store disks.
+	// +optional
+	InstanceStorePolicy *InstanceStorePolicy `json:"instanceStorePolicy,omitempty"`
```

Comment:
How do we handle the case where a user specifies `instanceStorePolicy: RAID0` but then also tries to mount the EBS volume on the same mount that the instance store is getting RAID0-ed and mounted on? Should we just disallow this case?

Reply:
Good point, not sure if there's a way to explicitly prevent that but it's probably worth calling out the mount point / device name in the docs

Reply:
Added this warning: https://github.com/aws/karpenter-provider-aws/blob/fdc8272f7c3ec8d7609dd56105433397b235511a/website/content/en/docs/concepts/nodeclasses.md?plain=1#L574-L576